### PR TITLE
gst-launch-1.0, gst-inspect-1.0: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/gst-inspect-1.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# gst-inspect-1.0
+
+> Print info on GStreamer plugins.
+> More information: <https://gstreamer.freedesktop.org/documentation/tools/gst-inspect.html?gi-language=c>.
+
+- Print info on a plugin:
```
Comment (kbdharun) [inline pages/common/gst-inspect-1.0.md:6]:
```suggestion
- Print information on a plugin:
```

File: pages/common/gst-inspect-1.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`gst-inspect-1.0 {{plugin_name}}`
```
Comment (kbdharun) [inline pages/common/gst-inspect-1.0.md:8]:
```suggestion
`gst-inspect-1.0 {{plugin}}`
```

File: pages/common/gst-inspect-1.0.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# gst-inspect-1.0
+
+> Print info on GStreamer plugins.
```
Comment (kbdharun) [inline pages/common/gst-inspect-1.0.md:3]:
```suggestion
> Print information on GStreamer plugins.
```

File: pages/common/gst-launch-1.0.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# gst-launch-1.0
+
+> Create gstreamer pipelines on the command line.
```
Comment (kbdharun) [inline pages/common/gst-launch-1.0.md:3]:
```suggestion
> Create GStreamer pipelines.
```

Comment (Managor) [reply pages/common/gst-launch-1.0.md:3]: I think the the "on the command line" is important as most of Gstreamer documentation is C code. As in "departing from the usual tools, here's a command line tool. What's opinion?

Comment (kbdharun) [reply pages/common/gst-launch-1.0.md:3]: We are currently moving away from specifying "CLI for", "on the command line", etc in page descriptions as it is generally understood that `tldr` is for CLI/console commands. For GUI  pages, we make distinctions in command descriptions instead.

But in this context, I think the presence of "on the command line." is fine as it provides a distinction from other components. @acuteenvy wdyt?

Comment (acuteenvy) [reply pages/common/gst-launch-1.0.md:3]: > I think the the "on the command line" is important as most of Gstreamer documentation is C code. As in "departing from the usual tools, here's a command line tool". What's opinion?

> But in this context, I think the presence of "on the command line." is fine as it provides a distinction from other components. @acuteenvy wdyt?

We're mostly documenting command-line tools here, maybe some GUI ones here and there, but never C libraries. All the "other components" are not documented in `tldr`. Therefore, explicitly stating that the page is about a command-line tool is redundant.

---

The official man page also doesn't say "on the command-line":
> `gst-launch-1.0` - build and run a GStreamer pipeline

```suggestion
> Build and run a GStreamer pipeline.
```

**Pull request merged**